src/app/projects/states/dashboard/directives/task-dashboard/directives/task-submission-card/task-submission-card.tpl.html

```diff
@@ -56,5 +56,26 @@ <h4>
         </li>
       </ul>
     </div><!--/more-dropdown-->
+    <br>
+    <div class="group-member-contribution-assessment">
+    <table class="table table-condensed table-hover">
+    <thead> 
+        <tr> 
+          <th> Comment </th>
+          <th> Type </th>
+          <th> Marker </th>
+          <th> Read Date </th>
+        </tr>                                             
+    </thead>
+    <tbody>
+      <tr ng-repeat="comment in task.comments">
```

Comment:
What about non-status comments?

Reply:
Hi Andrew, 
Thank you for your comment, I will work on that on next trimester.